R/opart_gaussian.R

```diff
@@ -43,10 +43,20 @@ opart_gaussian <- function(data, penalty) {
                data.vec = as.double(data),
```

Comment:
Hi @tdhock , I have removed the malloc declarations and passed the required temporary buffers through R code. Does this look ok?

Reply:
almost. 

please use `double(length(data))` instead of `as.double(vector(...))`

also why are you using `length(data)+1`? you should only need `length(data)`

Reply:
Hi @tdhock , I was using length(data) + 1 to keep things simple as then I don't have to worry about subtracting 1 from the index . I can change this to length(data).